Makefile

```diff
@@ -35,7 +35,7 @@ F90FLAGS = -module $(mod) -fpp -qopenmp -g -traceback -check bounds -debug exten
 
 # Let's start with setting up some paths
 # ---------------------------------------------------------------------------------------------------------------------------------------
-PATH_S3COM = $(HOME)/github/S3COM
+PATH_S3COM = $(HOME)/S3COM-test/S3COM
```

Comment:
Would be nice to find an easier solution than change the path back and forth between our commits

Reply:
Yes, I will rename my path to yours.

Reply:
Done!